src/js/component/App.js

```diff
@@ -20,7 +23,8 @@ class App {
     this.main = document.querySelector(settings.gridSection)
     this.nav = document.querySelector(settings.navSection)
     this.ul = document.querySelector(settings.ulSelector)
-    this.channelUILists = document.querySelector(settings.channelUIListsSelector)    
+    this.channelUILists = document.querySelector(settings.channelUIListsSelector)
```

Comment:
your view should take care of that

Reply:
https://www.sitepoint.com/mvc-design-pattern-javascript/ here is an example of how it usually used